packages/rancher-monitoring/generated-changes/overlay/templates/rancher-monitoring/dashboards/rancher/windows-dashboards.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if and .Values.grafana.enabled .Values.grafana.defaultDashboardsEnabled .Values.windowsExporter.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: {{ .Values.grafana.defaultDashboards.namespace }}
+  name: {{ printf "%s-%s" (include "kube-prometheus-stack.fullname" $) "windows" | trunc 63 | trimSuffix "-" }}
+  annotations:
+{{ toYaml .Values.grafana.sidecar.dashboards.annotations | indent 4 }}
+  labels:
+    {{- if $.Values.grafana.sidecar.dashboards.label }}
+    {{ $.Values.grafana.sidecar.dashboards.label }}: "1"
```

Comment:
What does this do, exactly?

Reply:
Creates a ConfigMap with the label `grafana_dashboards: 1` and any other user provided labels (which allows it to be picked up by the Grafana sidecar).

Then embeds all JSON files stored at files/rancher/windows/*